app/views/docs/certificates.phtml

```diff
@@ -9,12 +9,34 @@
 
 <h2>Debugging</h2>
 
-<p>In case you&#39;re still struggling with getting your certificate to work, check the Appwrite certificates worker log. You can do that with the following command:</p>
+<p>In case you're still struggling with getting your certificate to work, check the Appwrite certificates worker log. You can do that with the following command:</p>
```

Comment:
Suggested phrasing > In case you're still struggling to get your certificates working, 
Or consider simply > In case you're still struggling with your certificates

I also would consider using the word "if" instead of "in case", because "in case" can sound passive-aggressive, or that if their certs aren't working, they did something wrong.

Reply:
`If you're still struggling with your certificates, check the Appwrite certificates worker log.` applied